### PR TITLE
Meter v1

### PR DESCRIPTION
**Pull request by navin-moorthy**
Added Meter.

### CODE REVIEW

File: src/meter/MeterState.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {
+  UseProgressProps,
+  useProgressState,
+  ProgressStateReturn,
+} from "../progress";
+
+export type UseMeterProps = UseProgressProps;
+export const useMeterState = useProgressState;
+export type MeterStateReturn = ProgressStateReturn;
```
Comment (navin-moorthy) [inline src/meter/MeterState.ts:9]: Re-exporting the component and use them in storybook result in some warning.
I had to create a file and re export individually to remove those warnings.

**Comment by navin-moorthy** [comment]
Storybook changes are some quick refactors.

File: src/meter/MeterState.ts (hunk 1 above)
Comment (anuraghazra) [reply src/meter/MeterState.ts:9]: Okay i guess we can left it as it is.

File: src/breadcrumbs/BreadcrumbLink.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { createComponent, createHook } from "reakit-system";
 
-import { LinkHTMLProps, LinkOptions, useLink } from "../link";
+import { LinkHTMLProps, LinkOptions, useLink } from "../Link";
```
Comment (anuraghazra) [inline src/breadcrumbs/BreadcrumbLink.ts:3]: Linux systems does not care about case sensitive imports but windows does, so it was breaking the build.

**Comment by anuraghazra** [comment]
Okay i think git messed up the Case sensitive imports, we have to rename the Link file with git mv command in order to fix it.

**Pull request closed without merging**